simularium/simularium-viewer
# Fix/interface simularium file

Problem
=======
For the download option on the front end, the ISimulariumFile had no public way of getting the file contents back out. 

Solution
========
Created a `getAsBlob` function to get the file contents back out. We agreed this was a simple solution for the current implementation, but in the long run we want to create a writer class for getting the data back into a file format. 

with @toloudis and @interim17 

## Type of change

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. `npm start`
2.  load a trajectory (select one and click load)
3. click download
4. drag and drop the downloaded file back on the viewer 
5. click download again 
6. (can also check a binary file)
